### ISSUE TITLE
Makes  Extension host crash

### ISSUE THREAD

**Issue by jkohlin** [yzhang-gh/vscode-markdown#1142]
Issue Type: <b>Bug</b>

When creating a new extension from scratch (yo code) this extension caused extension host to exit: `Extension host with pid 35878 exited with code: 0, signal: null.`

Extension version: 3.4.3
VS Code version: Code 1.68.1 (30d9c6cd9483b2cc586687151bcbcd635f373630, 2022-06-14T12:48:37.209Z)
OS version: Darwin arm64 21.5.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|6, 10, 11|
|Memory (System)|64.00GB (0.67GB free)|
|Process Argv|config.yml --crash-reporter-id af5ad112-449c-437c-a6ef-0083a3e5bb1a|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdc:30486549

```

</details>

**Comment by yzhang-gh** [comment]
Thanks for the report. However, currently there isn't enough information for us to know what is going wrong. Feel free to comment when you have more details.

**Comment by jkohlin** [comment]
Well, I was just running [the getting started tutorial](https://code.visualstudio.com/api/get-started/your-first-extension) from vscode on how to create an extension and when I got to the point where I hit the F5 key, the extension host window just didn't open.
So I just ran the **Start Extension Bisect** command in vscode and it concluded that it was your extension that caused the trouble.
That's all the information I can provide I'm afraid.
